lib/fastlane/actions/mailgun.rb

```diff
@@ -26,13 +22,10 @@ def self.description
 
       def self.available_options
         [
-          FastlaneCore::ConfigItem.new(key: :mailgun_sandbox_domain,
```

Comment:
Yes, agreed with @joshdholtz. We could even proxy the values over to the new keys. The only problem is that it will show up multiple times in `fastlane action mailgun`